src/components/lime-ai-avatar/examples/lime-ai-avatar-colors.tsx

```
@@ -0,0 +1,54 @@
+ */
+@Component({
+    tag: 'limel-example-lime-ai-avatar-colors',
+    shadow: true,
+    styleUrl: 'lime-ai-avatar-basic.scss',
+})
+export class LimeAiAvatarColorsExample {
+    @State()
+    private isThinking: boolean = false;
+
+    @State()
+    private backgroundColor: string = 'rgb(var(--color-blue-default))';
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)